yk_gmd_blender/blender/exporter.py

```diff
@@ -277,7 +277,7 @@ def vertex_of(l, normal, tangent):
                         weights_list[3],
                     )
 
-                    v.uv = uv_blender_to_yk_space(l[uv_layer].uv)
+                    v.uv0 = uv_blender_to_yk_space(l[uv_layer].uv)
```

Comment:
If it's present, we should try to export UV1 as well as UV0

Reply:
Done.